Xamarin.Nuke.nuspec

```diff
@@ -10,7 +10,7 @@
     <authors>Jean-Marie Alfonsi, Tomasz Cielecki</authors>
     <owners>roubachof,cheesebaron</owners>
     <license type="file">LICENSE</license>
-    <projectUrl>https://github.com/roubachof/Xamarin.Nuke</projectUrl>
+    <projectUrl>https://github.com/roubachof/Xamarin.Nuke.NET6</projectUrl>
```

Comment:
The url is in fact https://github.com/roubachof/NukeProxy

Reply:
Yep, thanks. Fixed :)